relay-dynamic-config/src/defaults.rs

```diff
@@ -136,8 +138,9 @@ fn span_metrics(transaction_extraction_enabled: bool) -> impl IntoIterator<Item
         & RuleCondition::glob("span.op", "app.start.*")
         & RuleCondition::eq("span.description", APP_START_ROOT_SPAN_DESCRIPTIONS);
 
-    // `exclusive_time_light` is the metric with the most lenient condition.
+    // `exclusive_time_light` excludes transaction tags (and some others) to reduce cardinality.
     let exclusive_time_light_condition = (is_db.clone()
+        | is_ai.clone()
```

Comment:
Are we still interested in extracting this metric for AI spans?

Reply:
@Zylphrex

Reply:
The intention is to build a view similar to the index page of the various performance modules like database where it surfaces these ai metrics across the project.